### ISSUE TITLE
How to train MERS model?

### ISSUE THREAD

**Issue by HuangZuShu** [craston/MARS#45]
> Figure 2: Training to mimic the Flow stream. We first train the Flow stream to classify actions using optical flow clips with cross entropy loss and freeze its weights. To mimic flow features using RGB frames, in step 1, we backpropagate the MSE loss through all the layers of MERS except the last layer. In step 2, we separately train the last layer of MERS with a cross entropy loss.

Dear @craston , I am a llitle confused in your paper after looking your code .
According to your paper, I think  a model except the last layer has been trained with mse loss . And then you use this model to train the last layer with entropy loss. This is two separate process .
However, according to your code, you performance two steps in an epoch.
So the right operation is performancing two steps in an epoch ?
